Fixes #11316 - added note for tenant workaround (#1352)
MicrosoftDocs/azure-docs-powershell: azps-1.8.0/Az.Accounts/Connect-AzAccount.md

```

### -ContextName

> Name of the default context from this login. This context can be selected by this name

Reviewer: Need to change the term "login". Not sure what is being described here.

```

### -Credential

> Specifies a PSCredential object. For more information about the PSCredential object, type `Get-Help`

Reviewer: PSCredential should be bold

```

### -Tenant

Optional tenant name or ID

> [!NOTE]

> > Due to limitations of the current API, a tenant ID must be specified instead of a tenant name when
> > signing in with a business-to-business (B2B) account that has been assigned access to your Azure
> > tenant.

Reviewer: Passive voice

```suggestion
> Due to limitations of the current API, you must use a tenant ID instead of a tenant name when
> signing in with a business-to-business (B2B) account that has been assigned access to your Azure
> tenant.
```

```

### CommonParameters

This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,

> -WarningAction, and -WarningVariable. For more information, see about_CommonParameters
> (http://go.microsoft.com/fwlink/?LinkID=113216).

Reviewer: Fix the hyperlink

```suggestion
-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
